charts/mothership/templates/bundler.yaml

```
@@ -0,0 +1,79 @@
+{{- if .Values.bundler.enabled }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: stackup-bundler
+  namespace: {{ .Release.Namespace }}
+spec: 
+  selector:
+    matchLabels:
+      name: stackup-bundler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: stackup-bundler
+    spec:
+      initContainers:
+        - name: wait-for-op-geth
+          image: curlimages/curl
+          command: ['sh', '-c']
+          args:
+            - |
+              until curl -sf {{ $.Values.bundler.gethRpc }};
```

special thanks to ai-code-reviewer gpt,
```suggestion
              until curl -sf {{ .Values.bundler.gethRpc }};
```